### PR TITLE
[IMP]Improve the code.

### PR DESCRIPTION
**Pull request by KrishnaPrajapati-SerpentCS**

### CODE REVIEW

File: pos_order_for_restaurant/LICENSE/LICENSE [hunk 1]
```diff
@@ -0,0 +1,17 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2004-2010 OpenERP SA (<http://www.openerp.com>)
+    Copyright (C) 2011-2015 Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>).
```
Comment (JayVora-SerpentCS) [inline pos_order_for_restaurant/LICENSE/LICENSE]: 2011-TODAY .

File: pos_order_for_restaurant/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
```
Comment (JayVora-SerpentCS) [inline pos_order_for_restaurant/__init__.py]: Never write 'part of odoo'. The modules are not a part of odoo, right?

File: pos_order_for_restaurant/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
```
Comment (JayVora-SerpentCS) [inline pos_order_for_restaurant/__manifest__.py]: Remove part of odoo in all py files plz.

File: pos_order_for_restaurant/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+{
+    'name': 'Parcel/Delivery Order For Restaurant',
+    'version': '10.0.1.0.0',
+    'category': 'Point Of Sale',
+    'sequence': 6,
+    'summary': 'Touchscreen Interface for Restaurant',
+    'author': 'Serpent Consulting Services Pvt. Ltd.,\
+    Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
```
Comment (JayVora-SerpentCS) [inline pos_order_for_restaurant/__manifest__.py]: Remove the author OCA from all manifest files.

File: pos_order_for_restaurant/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    'application': True,
+    'data': ['security/pos_order_for_restaurant_security.xml',
+            'security/ir.model.access.csv',
+            'view/templates.xml',
+            'view/pos_order_for_restaurant_view.xml',
+           ],
+    'demo': [
+        'view/pos_order_restaurant_demo.xml',
+     ],
+    'depends': ['point_of_sale', 'pos_restaurant','pos_receipt'],
```
Comment (JayVora-SerpentCS) [inline pos_order_for_restaurant/__manifest__.py]: follow pep-8. ctrl+shift+F.

File: pos_order_for_restaurant/models/pos_order_for_restaurant.py [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+        final_list = []
+        if table_ids:
+            for table in self:
+                for table_user in table.users_ids:
+                    if table_user.id not in waiter_list:
+                        waiter_list.append(table_user.id)
+                        waiter_list_temp = {'id':table_user.id, 'name':table_user.name}
+                        final_list.append(waiter_list_temp)
+            return final_list
+
```
Comment (JayVora-SerpentCS) [inline pos_order_for_restaurant/models/pos_order_for_restaurant.py]: does not return need to be 4 spaces back?

File: pos_order_for_restaurant/models/pos_order_for_restaurant.py [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+            except AttributeError:
+                if not order.split_order :
+                    for res_table in order.reserved_table_ids:
+                        if (res_table.table_id.available_capacities - res_table.reserver_seat) == 0:
+                            table_obj.browse(res_table.table_id.id).write({'state' : 'available', 'available_capacities' : res_table.reserver_seat - res_table.table_id.available_capacities })
+                        else:
+                            if (res_table.table_id.available_capacities - res_table.reserver_seat) > 0:
+                                table_obj.browse(res_table.table_id.id).write({'state' : 'available', 'available_capacities': res_table.table_id.available_capacities - res_table.reserver_seat })
+        return res
+
```
Comment (JayVora-SerpentCS) [inline pos_order_for_restaurant/models/pos_order_for_restaurant.py]: please write comments on each block of code.

**Comment by KrishnaPrajapati-SerpentCS** [comment]
ok

File: pos_order_for_restaurant/__init__.py (hunk 2 above)
Comment (KrishnaPrajapati-SerpentCS) [reply pos_order_for_restaurant/__init__.py]: Yes

File: hotel/__manifest__.py [hunk 8]
```diff
@@ -1,34 +1,14 @@
...
-# ---------------------------------------------------------------------------
+# See LICENSE file for full copyright and licensing details.
 
 {
     'name': 'Hotel Management',
     "version": "10.0.1.0.0",
-    'author': 'Serpent Consulting Services Pvt. Ltd., OpenERP SA,\
-    Odoo Community Association (OCA),',
+    'author': 'Serpent Consulting Services Pvt. Ltd.,',
     'category': 'Generic Modules/Hotel Management',
```
Comment (JayVora-SerpentCS) [inline hotel/__manifest__.py]: you are not reading well, I did not say to remove openerp sa.

File: hotel/views/hotel_view.xml [hunk 9]
```diff
@@ -988,7 +944,7 @@
         <field name="name">Rooms by Category</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">product.category</field>
-        <field name="domain">[('parent_id','=',False),('isroomtype','=',True)]
+        	<field name="domain">[('parent_id','=',False),('isroomtype','=',True)]
         </field>
```
Comment (JayVora-SerpentCS) [inline hotel/views/hotel_view.xml]: Correct indentation

File: hotel/views/hotel_view.xml [hunk 10]
```diff
@@ -1000,7 +956,7 @@
         <field name="name">Amenities by Category</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">product.category</field>
-        <field name="domain">[('parent_id','=',False),('isamenitytype','=',True)]
+        	<field name="domain">[('parent_id','=',False),('isamenitytype','=',True)]
         </field>
```
Comment (JayVora-SerpentCS) [inline hotel/views/hotel_view.xml]: correct indentation

File: hotel/views/hotel_view.xml [hunk 11]
```diff
@@ -1012,7 +968,7 @@
         <field name="name">Services by Category</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">product.category</field>
-        <field name="domain">[('parent_id','=',False),('isservicetype','=',True)]
+        	<field name="domain">[('parent_id','=',False),('isservicetype','=',True)]
         </field>
```
Comment (JayVora-SerpentCS) [inline hotel/views/hotel_view.xml]: correct indentation

File: pos_folio/Doc/ChangeLog.txt [hunk 12]
```diff
@@ -0,0 +1,5 @@
+===============================================================================
+ Version Change Log (pos_folio)
+===============================================================================
+1.0.1 --> 4 April 2017 : Krishna Prajapati
+    * Developed new module pos_folio.
```
Comment (JayVora-SerpentCS) [inline pos_folio/Doc/ChangeLog.txt]: remove all changelog files.

**Pull request merged**